src/alire/alire-origins-deployers-source_archive.adb

```diff
@@ -160,11 +172,20 @@ package body Alire.Origins.Deployers.Source_Archive is
                --  the current dir at the end of the scope.
                Guard : Directories.Guard (Directories.Enter (Dst_Dir))
                  with Unreferenced;
+
+               --  Option required on Windows to force e.g. C: to mean
+               --  a local file location rather than the net host C.
+               --  Not available as standard on MacOS.
+               Force_Local_Option : constant String :=
```

Comment:
This should be a `String_Vector` that is wither `Empty_Vector` of contains one item, otherwise there will be an empty string in the vector of arguments.

Reply:
See new commit 87995eb